config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,19 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// BareMetalPlatformStatus holds the current status of the BareMetal infrastructure provider.
+type BareMetalPlatformStatus struct {
+	// apiVIP holds the Virtual IP (VIP) that can be used to contact the external OpenShift API
```

Comment:
I would clarify that this VIP is for a self-hosted load balancer in front of API servers, and remove the word "external"